### ISSUE TITLE
Results can be wrong with lucene METADATA {ignoreNullValues:false}

### ISSUE THREAD

**Issue by Ziink** [orientechnologies/orientdb#5257]
orientdb-community-2.1.5

If a lucene index is created on multiple fields and with METADATA {ignoreNullValues:false}, and if a field is null or missing, it causes problem with select.

```sql
create class Item
create property Item.title string
create property Item.summary string
create property Item.content string
create index Item.i_lucene on Item(title, summary, content) fulltext engine lucene METADATA {ignoreNullValues:false}
insert into Item set title = 'test', content = 'this is a test'


select * from Item where title lucene 'test'
select * from Item where [title, summary, content] lucene 'test'
```

The first select returns a result but the second does not. 

Note: This was working in 2.1.2

**Comment by robfrank** [comment]
Ok, I was able to replicate the problem and I made a fix. I'll wrote some tests to ensure that all is ok.

**Comment by Ziink** [comment]
Thanks.

**Comment by robfrank** [comment]
fix committed on develop and 2.1.x branch. You can wait the next hotfix release (2.1.6) or build the jar by your own.
